src/sonic-yang-models/tests/yang_model_tests/tests_config/bgp_device_global.json

```diff
@@ -0,0 +1,29 @@
+{
+    "BGP_DEVICE_GLOBAL_WITH_TSA_TEST": {
+        "sonic-bgp-device-global:sonic-bgp-device-global": {
+            "sonic-bgp-device-global:BGP_DEVICE_GLOBAL": {
+                "STATE":{
+                    "tsa_enabled": "true"
+                }
+            }
+        }
+    },
+    "BGP_DEVICE_GLOBAL_WITH_TSB_TEST": {
+        "sonic-bgp-device-global:sonic-bgp-device-global": {
+            "sonic-bgp-device-global:BGP_DEVICE_GLOBAL": {
+                "STATE":{
+                    "tsa_enabled": "false"
```

Comment:
Can you add a unit test for default value?

Reply:
Done